src/util/system.cpp

```diff
@@ -899,25 +782,31 @@ bool ArgsManager::ReadConfigFiles(std::string& error, bool ignore_invalid_keys)
         bool emptyIncludeConf;
```

Comment:
I can't comment on the exact line above (L897), but that comment refers to 'override args' and so should be changed in this PR.

I think the comment and bool name are pretty confusing, and could be made much clearer. Suggestion:

```
        // `-includeconf` cannot be included in the command line arguments except
        // as `-noincludeconf` (which indicates that no conf file should be used).
        bool use_conf_file{true};
        {
            LOCK(cs_args);
            const std::vector<util::SettingsValue>* const includes = util::FindKey(m_settings.command_line_options, "includeconf");
            if (includes) {
                // ParseParameters() fails if a non-negated -includeconf is passed on the command-line
                assert(util::SettingsSpan(*includes).last_negated());
                use_conf_file = false;
            }
        }
        if (use_conf_file) {
```